lib/package_json.test.ts

```diff
@@ -271,7 +271,7 @@ Deno.test("exports have default last", () => {
   assertEquals(Object.keys(result.exports), ["."]);
   assertEquals(Object.keys(result.exports["."]), ["import", "require"]);
 
-  // "default" must always be last: https://github.com/denoland/dnt/issues/228
+  // "types" must always be first: https://github.com/denoland/dnt/issues/228
```

Comment:
Is this a rule enforced by something? I think it was more accurate originally.

>"default" - the generic fallback that always matches. Can be a CommonJS or ES module file. This condition should always come last.

https://nodejs.org/api/packages.html#exports

Reply:
> "types" - can be used by typing systems to resolve the typing file for the given export. This condition should always be included first.

It's more important that "types" should be first because it matter for more than just `default`.

Reply:
Ha! Fair enough, missed that.

Reply:
Sorry, I just realized I was laser focused on types being first because I knew that was wrong and just re-read what the original issue was (it's late here). It would have been good to include both that types should be first and that default should be last. I've added it to my working tree and will include that information in a future update.